pygmt/src/wiggle.py

```diff
@@ -94,6 +104,24 @@ def wiggle(self, data=None, x=None, y=None, z=None, **kwargs):
     """
     kwargs = self._preprocess(**kwargs)  # pylint: disable=protected-access
 
+    if (fill_positive or fill_negative) and kwargs.get("G") is not None:
+        raise GMTInvalidInput("Use either fill_positive/fill_negative or color.")
+
+    if kwargs.get("G") is not None:
```

Comment:
We still need a test to make sure that a warning is raised when `color` is used.

Reply:
While you wrote this comment I worked on a corresponding test @seisman 👍